src/java/com/phenix/pct/CompilationWrapper.java

```diff
@@ -230,9 +230,10 @@ public void setXCode(boolean xcode) {
         compAttributes.setXCode(xcode);
     }
 
+    @Deprecated
     @Override
     public void setXCodeKey(String xcodeKey) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1133)